packages/cozy-harvest-lib/src/components/KonnectorUpdateLinker.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+
+import { Query, withClient } from 'cozy-client'
+import AppLinker from 'cozy-ui/react/AppLinker'
+import Button from 'cozy-ui/react/Button'
+
+const STORE_SLUG = 'store'
+
+const getKonnectorUpdateUrl = (appData = [], konnector = {}) => {
+  const storeApp = appData.find(
+    app => app.attributes && app.attributes.slug === STORE_SLUG
+  )
+  if (!storeApp) return null
+
+  const storeUrl = storeApp.links && storeApp.links.related
+  if (!storeUrl) return null
+
+  return `${storeUrl}#/discover/${konnector.slug}/install`
+}
+
+const KonnectorUpdateButton = ({
+  disabled,
+  isBlocking,
+  href,
+  label,
+  onClick
+}) => (
+  <Button
+    disabled={disabled}
+    className="u-m-0"
+    href={href}
+    icon="eye"
+    label={label}
+    onClick={onClick}
+    theme={isBlocking ? 'danger' : 'secondary'}
+  />
+)
+
+export class KonnectorUpdateLinker extends PureComponent {
+  render() {
+    const { client, label, isBlocking, konnector } = this.props
+    return (
+      <Query query={client.find('io.cozy.apps')}>
+        {({ data, fetchStatus }) => {
+          const isFetching = fetchStatus !== 'loaded'
+          const konnectorUpdateUrl = getKonnectorUpdateUrl(data, konnector)
+          const isReady = !isFetching && konnectorUpdateUrl
+          if (isReady) {
+            return (
+              <AppLinker slug="store" href={konnectorUpdateUrl}>
+                {({ href, onClick }) => (
+                  <KonnectorUpdateButton
+                    href={href}
+                    isBlocking={isBlocking}
+                    label={label}
+                    onClick={onClick}
+                  />
+                )}
+              </AppLinker>
+            )
+          } else {
+            return (
+              <KonnectorUpdateButton
+                disabled={true}
+                label={label}
+                isBlocking={isBlocking}
+              />
```

Comment:
I don't like this `KonnectorUpdateButton` abstraction. Adding it and adding this "big" if just to deal with 2 props don't sound good. I think we can do something like : 
```jsx
<KonnectorUpdateButton
{...isReady ? {href, onClick} : {disabled: true}} 
```